GAN/main.swift

```diff
@@ -0,0 +1,213 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import TensorFlow
+import Python
+
+// Import Python modules.
+let matplotlib = Python.import("matplotlib")
+let np = Python.import("numpy")
+let plt = Python.import("matplotlib.pyplot")
+
+// Turn off using display on server / Linux.
+matplotlib.use("Agg")
+
+// Some globals
+let epochCount = 10
+let batchSize = 32
+let outputFolder = "./output/"
+let imageHeight = 28, imageWidth = 28
+let imageSize = imageHeight * imageWidth
+let latentSize = 64
+
+func plot(image: Tensor<Float>, name: String) {
+    // Create figure.
+    let ax = plt.gca()
+    let array = np.array([image.scalars])
+    let pixels = array.reshape(image.shape)
+    if !FileManager.default.fileExists(atPath: outputFolder) {
+        try! FileManager.default.createDirectory(atPath: outputFolder,
```

Comment:
We aren't using Objective-C style formatting for function call arguments. Could you reformat this like the following?
```suggestion
        try! FileManager.default.createDirectory(
            atPath: outputFolder,
            withIntermediateDirectories: false,
            attributes: nil)
```